### PR TITLE
Make the spec harness verify what it only snapshotted

### PR DESCRIPTION
**Pull request by DZakh**
A spec records what the library does along several axes and, until now, each axis was only ever compared against the library. Nothing compared them against each other, so a spec could be internally contradictory and stay green. Six new checks put them side by side, and every one of them found something on its first run.

`packages/sury/src` is untouched: no library code changes, so no bundle or perf movement. `origin/main` is merged in, so this sits on top of #430.

## 1. CI runs `checkSpec`, not a copy of it

`spec_test.ts` re-listed the checks `pnpm spec check` makes, by hand, and the list had drifted. The operation matrix, `vs.zod`, and the jsonSchema round-trip type rules were exercised by nothing but a manual run, so a divergence in any of them reached `main` with the whole suite green. The per-spec test is now one call to `checkSpec` itself, which also picks up `ts.constructionError` centrally instead of needing a `skipIf` guard per check.

The first thing it did was fail, which is the point. On the pre-merge head the matrix rejected every `Blob` and `File` golden on the CI runner and none of them locally: `returned [object File], but the golden returned [object File]`. The comparison went through `isDeepStrictEqual`, and two `File`s differing only in `lastModified` are equal under Node 22 and not under Node 24 - and the golden writer omits `lastModified` on purpose, since it defaults to the moment the file is built. #430's `sameBlob`, which compares constructor, name and bytes and never asks `deepStrictEqual` about a `Blob`, is the fix, and merging it turned the job green. That bug was reachable the whole time; nothing in CI ran the matrix, so nobody met it.

## 2. An example input has to run

`recomputeGoldens` evaluated the input inside the same `try` that caught the operation, so an input that could not run was recorded as that example's `error` golden:

```yaml
typo:
  input: notDefiend
  error: notDefiend is not defined   # green forever, running nothing
```

Worse, `transpileModule` *repairs* what it cannot parse, so a missing brace never reached the evaluator at all:

```yaml
unbalanced:
  input: '{ a: "hello"'      # transpiles to a complete object literal
  output: '{ a: "hello" }'   # a passing golden for an input nobody wrote
```

The input is evaluated ahead of the recompute, and the transpile reports its first syntactic diagnostic instead of repairing.

## 3. An error golden names what threw

An error golden held the message alone, so a leaked `TypeError`, a `RangeError` out of a `Date`, or a foreign exception from user code in a custom codec read exactly like an intended Sury rejection:

```diff
-        error: boom
+        error: "Error: boom"
```

A `SuryError` still records its message alone: the message is the product surface, and stamping one constant class name onto every error golden in the suite would cost bytes on every line to say what the format already guarantees. One golden in the suite gains one.

This composes with `errorConstructor` from #430 rather than competing with it. That field is the opt-out an author reaches for *once they know* a message belongs to the platform; the prefix is what makes an unanticipated leak visible in the first place, so there is something to opt out of. An example declaring `errorConstructor` records the constructor and skips the prefix entirely.

## 4. The published JSON Schema is asked about the spec's own values

A spec pins the document it publishes and the values its parser accepts, and never put the two together. Ajv, in draft-07 with formats off, now asks the document about every accepted `parse` example.

Only that direction is gated. JSON Schema is allowed to describe a *wider* set than the parser does, since a refinement has no keyword, so a rejected input validating carries no information. A document that rejects what the parser accepts is one a consumer would use to turn away good input.

Nine such values, in six specs. Three of those specs hold a bug the goldens should not keep:

```ts
S.fromJSONSchemaOrThrow({ type: "array", prefixItems: [{ type: "string" }, { type: "number" }], items: false })
// parse(["a", 1])                     ->  ["a", 1]
// jsonSchema.input, as published      ->  { items: false, type: "array", prefixItems: [...] }
// a draft-07 validator, on ["a", 1]   ->  rejected
```

The default target is draft-07, where `prefixItems` means nothing and `items: false` forbids every item, so that document rejects every non-empty array the schema accepts. Two specs, both now carrying a `FIXME:`. The third is `fromjsonschema-allof-base-schema` (added by #428): its object arm strips `foo` before the `allOf` refinement reads it, so what `parse` returns fails its own `jsonSchema.output`. Its encode block already carried a FIXME for the same bug seen from the other side.

The remaining six are recorded divergences, not bugs: string length counted in JS characters against `minLength` counted in code points, and IEEE-754 against `multipleOf`'s exact division.

**Ajv rather than `S.fromJSONSchemaOrThrow` or the `vs.zod` equivalent.** Both of those read a document into another type system first, so a keyword the reader does not implement becomes a weaker schema and the check passes on a document no validator would have accepted. Reading a document is the thing under test, so the checker cannot be another reader of it.

## 5. The `vs` equivalent is run, not just typed

`vs.zod` pinned what the other library's *types* say and never ran it, so two libraries that agree on `string` and disagree on which strings were green either way. Accept-or-reject only: the value produced, the wording of a rejection, and which of several failures is reported are Sury's own.

68 examples across 23 specs read differently:

```ts
S.isoTime      // "23:59:60Z"    accepted (leap second)     z.iso.time()   rejected
S.uri          // "a b"          rejected (RFC 3986)        z.url()        accepted
S.number       // Infinity       accepted                   z.number()     rejected
S.integer      // 1e+21          accepted                   z.int()        rejected
S.string.with(S.to, S.number)  // ""  rejected              z.coerce...    accepted (0)
```

Read through Standard Schema (`~standard`), not a Zod API, for the reason `deriveVsTypeInfo` reads the types that way: a `vs.valibot` would need only the import line and a key in the format.

## 6. Three smaller rules

- **Two examples with the same input** are one case with two names, so one of them covers nothing. Six, in four specs, now consolidated.
- **An example that answers differently each time** cannot have a golden. Previously this surfaced as a `--write` that rewrote the file every run and a staleness diff naming the schema.
- **`S.global` in a spec** sets process-wide configuration, and one library instance is shared by every spec in a run, so the failure lands on some other spec and only sometimes. The check reports it and refuses to evaluate the schema at all, which is the only way the rule can hold.

## Recording a divergence

Where a verifier legitimately disagrees with `parse`, the example says so - declared by the author, kept fresh by `--write`, reported when it stops being true. One field, named for the verifiers a reader can go and run, holding what each of them actually answered, and a reason that is required, because an answer without one says nothing anybody can act on:

```yaml
emoji:
  input: '"😀"'
  output: '"😀"'
  divergence:
    reason: minLength counts code points and S.minLength counts JS characters, so the emoji is one short of the document's bound
    ajv: "input: data must NOT have fewer than 2 characters; output: data must NOT have fewer than 2 characters"
```

`check` and `ajv` build no output, so `true` is the whole of what they say when they accept; anything else is the message they reject with. `zod` has no `true` - it always has words of its own, so it records the value it returns, as source, or its rejection:

```yaml
divergence:
  reason: the equivalent is a bare transform with nothing validating its result, and Number("") is 0 while Number("abc") is NaN
  zod: NaN
```

The answer, not a `passes`/`fails` marker, because the answer is the half a reader wants and the half that quietly changes under a refactor. `reason` is the author's and `--write` never touches it; the verdicts beside it stay fresh the way `creationError` does. The shape carries three rules of its own, each with a test: a blank reason, a `divergence` naming no verifier, and one recorded on `decode` or `encode`, where a verifier disagreeing with an operation that trusts its input is by design.

The gate around all three stays accept-or-reject: two libraries that both accept an input and hand back different values still count as agreeing. Comparing `divergence.zod` against the example's own `output` is the obvious next step and is logged under **Spec Harness Suggestions** rather than built here - it would turn every coercion difference into a finding, and now that the answers are recorded the size of that is measurable before deciding.

## The PR performance comment

The comment had no verdict, sorted every row worst-first into one table so a change with eleven regressions truncated away every improvement it also made, and left the targets it could not time in `<sub>` small print next to the node version.

```
### Spec performance

**2 slower, 1 faster** of 140 timed targets · `83f943b` vs `93999e3` (origin/main)

**Not timed**

- behavior changed, not timed - optional-object · parse · array-is-not-an-object: baseline accepted it, now rejected
- could not measure union5 · create: boom

**Slower** / **Faster**  (separate tables, each truncating on its own)
```

It still parses the CLI's own text rather than adding a `--json` mode, so the artifact and the comment cannot disagree.

## Coverage

`packages/spec` is now measured. It is test infrastructure, but it is also what decides whether 478 specs pass, so an unexercised branch in it is a check nobody is running. Two things had to change before it reported anything: a `../spec/**` pattern is resolved against `packages/sury` and matches nothing, and v8 drops files outside the project root without `allowExternal`.

Separately, the run now writes `lcov.info`. The default reporters (`text`, `html`, `clover`, `json`) never did, so the codecov step that points at `packages/sury/coverage/lcov.info` had nothing to upload.

`index.mjs` rather than `src/**/*.ts` in the include: the tests import the bundle, so naming the sources reports every one of them at 0%.

## Impact

```
bundle   unchanged - packages/sury/src is untouched
perf     unchanged - no library code changed

gates    1141 tests · 478 specs · tsc clean · knip clean
         coverage  90.4% statements overall
                   84.0% statements / 78.2% branches for packages/spec
                   bench.ts at 46% is the floor - a child process driven by
                   wall-clock measurement

deps     ajv 8.20.0 and zod 4.4.3, devDependencies of packages/spec only
```

Merging #430 rewrote no goldens: all 478 specs passed against the new codegen unchanged, and the divergences below survived it.

## Behavior changes

- 39 specs gain a `divergence` on an example - 101 in all, each with a hand-written reason and the verifier's own answer, filled by `--write`. No parse golden's value changed.
- One error golden gains its class (`string-to-blob`, `Error: boom`).
- Six duplicate examples are removed and two example names are widened to say what the single remaining case covers.
- Three `FIXME:` comments are added for the bugs in §4.
- `spec.schema.json` gains the `divergence` object.

## Also

- The `spec` skill drops the harness internals it had drifted into repeating - the list of cross-checks and their marker names, what a scenario measures inside - and says what it is for: how to run the CLI, and to do what its messages say. It now names no field the format can rename, which is what let the old text go stale.
- The ASCII box in `CONTRIBUTING.md` lines up again; its right border stopped matching when #436 renamed `expected.pr` to `expected.parser`.
- `tests/spec_errors_test.ts` gains 12 tests and `spec_test.ts` two, one per new message, so each is proved to guide rather than just fail.
- A review round closed four holes the new rules left open. The largest: `serialize` evaluates an example input while canonicalizing it (`reformatIfEvaluable`), so the `S.global` guard had to move ahead of the canonical-form check, and the lint had to read example inputs and not just `ts.schema`. A scenario that calls it is now skipped rather than merely reported, since building one runs it. The threads carry the detail.
- `CONTRIBUTING.md` loses four bullets. Two by request: the Result-mode one (#430 built `resultExpression`) and the Blob/File one (stale, since the golden writer reads a container's bytes and 11 specs carry such outputs). Two more because #430 built them: `ts.constructionError`, and the "compare the class alone" ending of the platform-wording bullet, which now points at `errorConstructor` instead. The `assert`/`is` bullet keeps its ask, since the slots now exist and no spec fills one. Six new bullets are added, including the two things this deliberately does not check: a document *wider* than the parser passes silently, and so does a `vs.zod` equivalent that accepts the same inputs but returns different values.
- A stale comment in `tests/content_test.ts` repeating the Blob claim is corrected.

---
🤖 Generated with [Claude Code](https://claude.com/claude-code)

https://claude.ai/code/session_01WHR4LPSYv9DcfdCA9UpMYE

## Summary by CodeRabbit

* **New Features**
  * Added JSON Schema validation for specification examples.
  * Added structured divergence reporting for checks, JSON Schema, and Zod results, including detailed explanations and verifier responses.
  * Added coverage reporting and improved performance summaries, including untimed and faster/slower results.

* **Bug Fixes**
  * Global configuration scenarios are now reported and skipped safely.
  * Error comparisons now require matching error class prefixes and messages.

* **Documentation**
  * Updated specification and contribution guidance for revised validation and divergence workflows.

### CODE REVIEW

File: packages/spec/harness.ts [hunk 1]
```diff
@@ -196,6 +198,25 @@ export const lintExamples = (spec: Spec, out: string[]): void => {
...
+// shared by every spec in a run (cli.ts checks them under Promise.all, and
+// spec_test.ts in one Vitest worker). A spec that calls it changes what its
+// neighbours compile, in an order nothing controls - so the failure surfaces on
+// some other spec, intermittently, and never on the one that caused it.
+const GLOBAL_CALL = /\bS\s*\.\s*global\b/;
+export const lintGlobal = (spec: Spec, out: string[]): void => {
+  const sources: [string, unknown][] = [["ts.schema", spec.ts?.schema]];
+  const aliases = spec.ts?.aliases;
+  if (Array.isArray(aliases))
+    aliases.forEach((alias, i) => sources.push([`ts.aliases[${i}]`, alias]));
```
[the remark this answers is not in the document]
Comment (DZakh) [reply packages/spec/harness.ts:211]: Fixed in 1f41be7, and the hole was deeper than reported.

`lintGlobal` now reads every `operations.*.examples.*.input`. But the evaluation happens earlier than `checkExamples`: canonicalizing an example reformats its input by running it (`reformatIfEvaluable`), and `checkSpec` builds the canonical form before its early return, so the call had already executed by the time the guard fired. The return now sits immediately after the lint, and such a spec gets no canonical-form or golden report at all, which is the point.

Writing the regression test is what surfaced that. My first attempt built its fixture with `serialize`, which ran the `S.global` call and rewrote the input to its value, so the test saw nothing. It splices the example into the YAML text instead.

---
_Generated by [Claude Code](https://claude.ai/code)_

File: packages/spec/harness.ts [hunk 2]
```diff
@@ -1592,6 +1802,197 @@ export const checkOperationMatrix = async (spec: Spec, schema: any): Promise<str
...
+      else byInput.set(ex.input, exName);
+
+      // Run twice on two fresh values. A golden derived from the clock, from
+      // `Math.random`, or from iteration order that is not stable would be
+      // rewritten by every `--write` and reported as a staleness diff on every
+      // check, naming the schema instead of the reason.
+      if (!("fn" in built)) continue;
+      const first = await runToRef(built.fn, nextData());
+      const second = await runToRef(built.fn, nextData());
+      if (!(await sameOutcome(first, second)))
```
[the remark this answers is not in the document]
Comment (DZakh) [reply packages/spec/harness.ts]: Fixed in 1f41be7. The determinism check has its own comparison now: exact equality for a failure, `sameOutcome` for a value so a `Blob` is still compared by its bytes rather than by identity.

You are right about why the borrowed one was wrong. `sameMessage` spans `boom` and `Error: boom` because a golden and a Result spelling write one failure two ways, which is correct there and not here.

No regression test for this one: a `SuryError` whose message exactly equals a foreign error's is not constructible through the public API without contortion, so the case the relaxed matching covered has no reachable reproduction. The fix is strictly tighter either way.

---
_Generated by [Claude Code](https://claude.ai/code)_

File: packages/spec/harness.ts [hunk 3]
```diff
@@ -1697,6 +2127,15 @@ export const checkScenarios = (
...
       errs.push(`${id}: invalid scenario id (only letters, digits, and - allowed)`);
+    for (const [field, source] of [
+      ["prepare", scenario.prepare],
+      ["run", scenario.run],
+    ] as const)
+      if (typeof source === "string" && GLOBAL_CALL.test(source))
+        errs.push(
+          `${id}: ${field} calls S.global - it sets process-wide configuration that every spec and ` +
+            "scenario in the run then compiles against",
+        );
```
[the remark this answers is not in the document]
Comment (DZakh) [reply packages/spec/harness.ts]: Fixed in 1f41be7. `checkScenarios` collects the offending fields, reports each, and `continue`s before `buildScenarioRunner`, which as you say runs the scenario once while constructing it.

Covered by a test in `spec_test.ts`. It asserts the report is the only error, so the absence of a `did not run` entry is also the evidence that nothing executed.

---
_Generated by [Claude Code](https://claude.ai/code)_

File: packages/spec/perfComment.ts [hunk 4]
```diff
@@ -33,34 +60,58 @@ export const renderComment = (
...
 
+  const slower = rows.filter((r) => r.dir === "slower");
+  const faster = rows.filter((r) => r.dir === "faster");
+  const unchanged = Number(lines.join("\n").match(UNCHANGED)?.[1] ?? NaN);
+  const measured = Number.isNaN(unchanged) ? undefined : unchanged + rows.length;
+
   const out = [`### ${heading}`, ""];
+
+  if (!header) {
+    out.push("_report could not be parsed - see the full report below._", "");
```
[the remark this answers is not in the document]
Comment (DZakh) [reply packages/spec/perfComment.ts]: Fixed in 1f41be7, with the conditional you proposed. The branch is reachable: the drift job renders into a run summary with no artifact uploaded, so that wording pointed at nothing.

I checked both renderings rather than only the changed one.

---
_Generated by [Claude Code](https://claude.ai/code)_

File: .claude/skills/spec/SKILL.md [hunk 5]
```diff
@@ -17,6 +17,8 @@ Add a case: a named entry with just `input` under an op's `examples`, then `--wr
 
 **Examples are where findings live.** Cover every edge case the schema turns up - boundary values, IEEE-754 oddities (`-0`, `NaN`, `Infinity`), coercion corners, each generated-check branch. A bug report or review finding becomes an example, not a test file and not a commit message.
 
+Every example is then re-run by every cross-check the harness makes: the other spellings of its own operation, the `jsonSchema` the same spec publishes (with a real validator), and the `vs.zod` equivalent. All three are expected to agree with `parse`, so where one genuinely does not, the example records it - `whenChecked`, `whenValidated`, `whenZod`. The check names the marker and the value to write. Add one only when the disagreement is real behaviour; if it is a bug, add the marker *and* a `FIXME:` comment saying so.
```
Comment (DZakh) [inline .claude/skills/spec/SKILL.md]: Drop implementation details from the skill - clean up other parts as well, just make it to teach how to run spec cli

File: packages/spec/format.ts
Comment (DZakh) [inline packages/spec/format.ts]: For all the cases instead of passes/fails let's have something like:

```
divergence?:
  reason: "free-text reasno"
  zod?: "actual result if different",
  ajv?: "actual result if different",
  check?: "actual result if different",
 ```

Comment (DZakh) [reply packages/spec/format.ts]: Done in 9631f48, in the shape you wrote. One `divergence` per example, named for the verifiers a reader can go and run:

```yaml
emoji:
  input: '"😀"'
  output: '"😀"'
  divergence:
    reason: minLength counts code points and S.minLength counts JS characters, so the emoji is one short of the document's bound
    ajv: fails
```

Two things I decided rather than asked about, say the word if either is wrong:

- The verdicts stayed `passes`/`fails`. All three verifiers answer accept-or-reject and nothing else: the value produced and the wording of a rejection are Sury's own and are deliberately never compared, so there is no other "actual result" for one to record.
- `reason` is required. That was the point of the change, so the field the author writes is not the optional one.

`reason` is never touched by `--write`; the verdicts beside it stay fresh the way `creationError` does. Three rules the shape now carries, each with a test in `spec_errors_test.ts`: a blank reason, a `divergence` naming no verifier, and one recorded on `decode` or `encode`, where a verifier disagreeing with an operation that trusts its input is by design.

All 101 markers across 39 specs are migrated with a reason written out, drawn from what the check itself reports. The 22 `whenChecked: fails` were nearly all one fact worth stating once:

```yaml
divergence:
  reason: the checks compile the reverse direction, which this codec leaves ambiguous, so they raise on the schema instead of answering about the value
  check: fails
```

---
_Generated by [Claude Code](https://claude.ai/code)_

File: .claude/skills/spec/SKILL.md (hunk 5 above)
Comment (DZakh) [reply .claude/skills/spec/SKILL.md]: Rewritten in 9631f48. The paragraph you flagged is gone, along with the rest of the harness internals the skill had drifted into repeating.

Out: the list of cross-checks and what each one re-runs, the marker names and when to add which, what a scenario measures internally ("the dispatch around a compiled operation, invisible to every per-spec phase"), and the instruction to log gaps in `CONTRIBUTING.md`, which is a rule for whoever maintains the harness rather than for someone running it.

In: `pnpm spec schema`, which the skill never mentioned and which `check` tells you to run, and one rule in place of all of the above - follow the CLI's messages, since every rule the format has reports its own fix where it is broken. That is what keeps the skill from going stale the way this paragraph did: it names no field the format can rename.

What stayed: the commands, that a finding becomes an example rather than a test file, the metrics ratchet and that its summary is the deliverable, the noise floor, and scenarios.

---
_Generated by [Claude Code](https://claude.ai/code)_

File: .claude/skills/spec/SKILL.md [hunk 6]
```diff
@@ -5,27 +5,52 @@ description: Develop Sury with the spec CLI. Use whenever changing Sury core log
 
 # Sury specs
 
-One `specs/<id>.yaml` = one schema's contract: type, JSON Schema, per-operation codegen + examples. You author the schema, `ts.aliases`, `vs.zod`, and example *inputs*. **Never hand-write a golden** - `pnpm spec` derives every one.
+One `specs/<id>.yaml` = one schema's contract: its type, its JSON Schema, and per
```
[the remark this answers is not in the document]
Comment (DZakh) [reply .claude/skills/spec/SKILL.md]: Fixed in ee222e6. Two of the three are qualified, and the third is gone rather than qualified.

Lines 8 and 51 now read `packages/sury/specs/<id>.yaml` and `packages/sury/specs/scenarios.yaml`. Line 17 was a comment on a command, `# regenerate specs/spec.schema.json`, and a path there earns its keep only if the reader needs it - they do not, since it is that command's own output. It reads `# regenerate the specs' JSON Schema`, and Layout gained the file along with what it is for: what an editor reads for hover and completion.

Both rewrapped lines are back under the file's 80 columns.

---
_Generated by [Claude Code](https://claude.ai/code)_

File: packages/spec/format.ts
Comment (DZakh) [reply packages/spec/format.ts]: Done in 346fd62 - the verdicts are the real answer now, not `passes`/`fails`.

```yaml
divergence:
  reason: the equivalent is a bare transform with nothing validating its result
  zod: NaN
```

`check` and `ajv` build no output, so `true` is the whole of what they say when they accept; anything else is the message they reject with - the validator's own words, per side, for `ajv`:

```yaml
reason: minLength counts code points and S.minLength counts JS characters, so the emoji is one short of the document's bound
ajv: "input: data must NOT have fewer than 2 characters; output: data must NOT have fewer than 2 characters"
```

`zod` has no `true`: it always has words of its own, so it records the value it returns, as source, or its rejection. `uri` reads well now - the answers are the URLs `z.url()` hands back for inputs `S.uri` turns away.

All 101 entries were refilled by `--write` rather than by hand, the guiding messages carry the value to write (`record it with divergence: { reason: ..., zod: "null" }`), and a recorded answer that stops matching is reported against the one that replaced it.

One thing this opens that I did not build, logged under Spec Harness Suggestions instead: the gate is still accept-or-reject, so two libraries that both accept an input and hand back different values still count as agreeing. Comparing `divergence.zod` against the example's own `output` would make every coercion difference a finding - the reason not to today is noise, not principle, and now that the answers are recorded the size of that noise can actually be measured first.

Verified: 478 specs green, 1141 tests, tsc and knip clean.

---
_Generated by [Claude Code](https://claude.ai/code)_

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -399,6 +383,46 @@ instead of silently working around it.
...
   cause: a no-op accepts every input, valid or not.
+
+- `divergence.zod` now records the value the equivalent returns, but the gate
+  around it is still accept-or-reject: two libraries that both accept an input
+  and hand back different values agree as far as the check is concerned, and
+  `divergence.zod agrees with parse - remove it` is what an author who recorded
+  the difference is told. Comparing that value against the example's own
+  `output` would turn every coercion difference into a finding - the reason it
+  is not done today is noise, not principle, and the answers are now recorded,
+  so the size of that noise can be measured before deciding.
```
[the remark this answers is not in the document]
Comment (DZakh) [reply CONTRIBUTING.md]: Right, and the bullet was mine from an hour ago. Fixed in 173e00c.

I checked it against the code rather than taking it as read. `checkZodExamples` compares a recorded `zod` answer against the live one and reports the difference (`divergence.zod records "x" but the vs.zod equivalent answers "y"`), so a stale entry does fail the gate. What is accept-or-reject is the decision that a divergence exists at all: when both libraries accept, nothing is recorded, and a value difference there is invisible.

So the bullet now names that case instead of the whole field, and says what closing it would cost:

> What decides whether a `divergence` exists at all is accept-or-reject, so an example both libraries accept records nothing, and a value difference there is invisible. Once one exists its `zod` answer is a golden like any other - a changed value is reported against the one it replaced - but an author who notices the difference first and records it is told `divergence.zod agrees with parse - remove it`.

---
_Generated by [Claude Code](https://claude.ai/code)_
